network_files/config/genesis.json

```diff
@@ -0,0 +1,3494 @@
+                "commission": {
+                  "rate": "0.100000000000000000",
+                  "max_rate": "0.200000000000000000",
+                  "max_change_rate": "0.010000000000000000"
+                },
+                "min_self_delegation": "1",
+                "delegator_address": "zeta1p3emgemv8q0fmtw70kfzwecmcvyd9ztqmzy3r9",
+                "validator_address": "zetavaloper1p3emgemv8q0fmtw70kfzwecmcvyd9ztqlzudwn",
+                "pubkey": {
+                  "@type": "/cosmos.crypto.ed25519.PubKey",
+                  "key": "2fe92qJjVA9yTdu/e0dgQieIorL8C6HRo8Fj0VuINP0="
+                },
+                "value": {
+                  "denom": "azeta",
+                  "amount": "1000000000000000000000"
+                }
+              }
+            ],
+            "memo": "7b1e4c6dbdeb65fbaafd5fe415679d252bdae2f9@141.94.214.137:26656",
+            "timeout_height": "0",
+            "extension_options": [],
+            "non_critical_extension_options": []
+          },
+          "auth_info": {
+            "signer_infos": [
+              {
+                "public_key": {
+                  "@type": "/cosmos.crypto.secp256k1.PubKey",
+                  "key": "AzF0ZCuTvpGK22L180MLctss2wc4Ao4tL094d8R3Pkv+"
+                },
+                "mode_info": {
+                  "single": {
+                    "mode": "SIGN_MODE_DIRECT"
+                  }
+                },
+                "sequence": "0"
+              }
+            ],
+            "fee": {
+              "amount": [],
+              "gas_limit": "200000",
+              "payer": "",
+              "granter": ""
+            },
+            "tip": null
+          },
+          "signatures": [
+            "4V6yhbgOgLSVd5YPG5RbPnEZt8eyN8+vJSF/KsJQ015uOR4/RQL+KktePDytgpqw3jzsWYFR7ymbcFIl1QkbCg=="
+          ]
+        },
+        {
+          "body": {
+            "messages": [
+              {
+                "@type": "/cosmos.staking.v1beta1.MsgCreateValidator",
+                "description": {
+                  "moniker": "bastion",
+                  "identity": "",
+                  "website": "",
+                  "security_contact": "",
+                  "details": ""
+                },
+                "commission": {
+                  "rate": "0.100000000000000000",
+                  "max_rate": "0.200000000000000000",
+                  "max_change_rate": "0.010000000000000000"
+                },
+                "min_self_delegation": "1",
+                "delegator_address": "zeta1g323lusfa9qqvjvupajre2dphuem999fahc086",
+                "validator_address": "zetavaloper1g323lusfa9qqvjvupajre2dphuem999fehqn2v",
+                "pubkey": {
+                  "@type": "/cosmos.crypto.ed25519.PubKey",
+                  "key": "gpc90UewDCIMDOGOP7FFC7iweEiuZgOQ791Fi/5NDh8="
+                },
+                "value": {
+                  "denom": "azeta",
+                  "amount": "1000000000000000000000"
```

Comment:
All the amounts are the same instead of
- [ ]  Initial Bonds:
    - [ ]  OS: 4,200,000
    - [ ]  Core: 100,000-200,000k